man/systemd-system.conf.xml

```diff
@@ -438,17 +438,21 @@
         <term><varname>DefaultIOAccounting=</varname></term>
         <term><varname>DefaultIPAccounting=</varname></term>
 
-        <listitem><para>Configure the default resource accounting settings, as configured per-unit by
+        <listitem>
+        <para>Configure the default resource accounting settings, as configured per-unit by
         <varname>CPUAccounting=</varname>, <varname>MemoryAccounting=</varname>,
         <varname>TasksAccounting=</varname>, <varname>IOAccounting=</varname> and
         <varname>IPAccounting=</varname>. See
         <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-        for details on the per-unit settings. <varname>DefaultTasksAccounting=</varname> defaults to yes,
-        <varname>DefaultMemoryAccounting=</varname> to &MEMORY_ACCOUNTING_DEFAULT;.
-        <varname>DefaultCPUAccounting=</varname> defaults to yes, but really has no effect if enabling CPU
-        accounting doesn't require the <option>cpu</option> controller to be enabled (Linux 4.15+ using the
-        unified hierarchy for resource control), otherwise it defaults to no. The other three settings
-        default to no.</para></listitem>
+        for details on the per-unit settings.</para>
+
+        <para><varname>DefaultCPUAccounting=</varname> defaults to yes, but really has no effect if enabling
+        CPU accounting doesn't require the <option>cpu</option> controller to be enabled (Linux 4.15+ using
+        the unified hierarchy for resource control), otherwise it defaults to no.
```

Comment:
actually the comment about the kernel should be removed. this actually has an affect, as our own apis to determine cpu use are not available if CPU accoutning is not enabled for a unit. please drop the sentence altogether.

Reply:
"`DefaultCPUAccounting=` defaults to `yes` with Linux >= 4.15 and the unified CGroup hierarchy, otherwise `no`" works for me if accurate, but I'm not sure if I'm right... :)

Reply:
the unified hierarchy is already depreacted from our side, we stopped documenting it. hence please no mention of it here.

Reply:
Got it, thanks

Reply:
Oh for some reason I didn't see your replies to my original comment. So I was talking to myself. I'll push a new commit that mentions the kernel version shortly.